python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_p2p_links.schema.yml

```diff
@@ -188,6 +188,22 @@ $defs:
               type: str
               valid_values: ["on", "active", "passive"]
               default: active
+            channel_id_generation:
```

Comment:
The key name has been nagging me. Can we do `channel_id_algorithm` to be more in line with what we have elsewhere?

Reply:
I changed it to channel_id_algorithm.